tests/checkBoxes.spec.ts

```diff
@@ -1,30 +1,28 @@
 import { test, expect } from '@playwright/test';
 
-
 test.describe('Input fields', () => {
     test.beforeEach(async ({ page }) => {
         await page.goto('/')
         await page.getByRole('button', { name: 'Veterinarians' }).click()
         await page.getByRole('link', { name: 'All' }).click()
-
     })
 
-
-
     test('Test Case 1: Validate selected specialties', async ({ page }) => {
         await expect(page.getByRole('button', { name: 'Veterinarians' })).toHaveText('Veterinarians');
         await page.getByRole('row', { name: "Helen Leary" }).getByRole('button', { name: "Edit Vet" }).click()
         await expect(page.locator(".selected-specialties")).toHaveText('radiology')
         await page.locator('.dropdown-arrow').click()
-        expect(await page.getByRole('checkbox', { name: "radiology" }).isChecked()).toBeTruthy()
-        expect(await page.getByRole('checkbox', { name: "surgery" }).isChecked()).toBeFalsy()
-        expect(await page.getByRole('checkbox', { name: "dentistry" }).isChecked()).toBeFalsy()
-        await page.getByRole('checkbox', { name: "surgery" }).check()
-        await page.getByRole('checkbox', { name: "radiology" }).uncheck()
+        const checkBoxRadiology = page.getByRole('checkbox', { name: "radiology" })
+        const checkBoxSurgery = page.getByRole('checkbox', { name: "surgery" })
+        const checkBoxDentistry = page.getByRole('checkbox', { name: "dentistry" })
+        expect(await checkBoxRadiology.isChecked()).toBeTruthy()
+        expect(await checkBoxSurgery.isChecked()).toBeFalsy()
+        expect(await checkBoxDentistry.isChecked()).toBeFalsy()
+        await checkBoxSurgery.check()
+        await checkBoxRadiology.uncheck()
         await expect(page.locator(".selected-specialties")).toHaveText('surgery')
-        await page.getByRole('checkbox', { name: "dentistry" }).check()
+        await checkBoxDentistry.check()
         await expect(page.locator(".selected-specialties")).toHaveText('surgery, dentistry')
```

Comment:
I think I was not clear enough in my previous comment.
I mean the selector '.selected-specialties' was used 3 times in the code so it makes sense to extract the locator to a constant.
As for other locators related to the checkboxes, technically it's up to you. For me it was fine the way it was, because the locator itself was very readable and was used only twice. But if you want to use it as a constant - it's also fine.